Interlaminar Analysis

JessyD opened the pull request:
**1. Comparison of the Power Spectrum of Layer 2/3 and Layer 5/6 generated by the Python and NeuroML/LEMS simulation**
![screen shot 2018-11-12 at 15 54 46](https://user-images.githubusercontent.com/10345440/48359307-94280880-e694-11e8-85e2-e5d7884e0209.png)
(While top two images represent the power spectrum of the Python simulation, the bottom show the power spectrum generated with NeuroML/LEMS simulation)

**2. Analysis of the Rate over time for the Python and NeuroML/LEMS Simulation**
![screen shot 2018-11-12 at 15 54 34](https://user-images.githubusercontent.com/10345440/48359419-d5201d00-e694-11e8-8229-06eb0bc07b6b.png)
This images can be generated by calling: `python GenerateNeuroMLlite.py -interlaminar`

**Periodogram of layer 2/3 from the Python simulation**
![periodogram](https://user-images.githubusercontent.com/10345440/48359581-2c25f200-e695-11e8-8d4a-510a35194d8f.jpg)
To generate this image I have simulated the interlaminar rate and analysed it with the Python functions that I have defined and saved the intermediate results as a pickle (`periodogram.pckl`). I then load the results on Matlab and use the closed sourced function to analyse it.

pgleeson commented:
@JessyD I just zoomed out on the NML trace (left)...

![image](https://user-images.githubusercontent.com/1556687/48421158-d2383180-e753-11e8-9587-29ec874051ba.png)

It's an issue with ms & seconds...

JessyD commented:
@pgleeson 
I have fixed the initial rate of the simulation to be always 5 Hz. This avoids the initial spike on the Python.
![screen shot 2018-11-23 at 18 35 10](https://user-images.githubusercontent.com/10345440/48957310-ae44cf00-ef4f-11e8-9fc3-6b29b41bc2df.png)

I have also calculated the power spectrum of the Python (top) and NeuroML (bottom) using the `spectral.compute_spectrum` from the `neurodsp` package.
<img width="1274" alt="screen shot 2018-11-23 at 18 45 00" src="https://user-images.githubusercontent.com/10345440/48957406-5a86b580-ef50-11e8-9c40-5132d9bf92c1.png">
![screen shot 2018-11-23 at 18 52 01](https://user-images.githubusercontent.com/10345440/48957524-e862a080-ef50-11e8-8fb4-3bcd34da5aa0.png)

And I fixed the issue with the ms & seconds for the NeuroML/LEMS simulation. (Python simulation (top), NeuroML/LEMS (bottom))
<img width="1271" alt="screen shot 2018-11-23 at 18 45 20" src="https://user-images.githubusercontent.com/10345440/48957452-8f930800-ef50-11e8-8800-ce08475f4cab.png">
![screen shot 2018-11-23 at 18 51 46](https://user-images.githubusercontent.com/10345440/48957550-17791200-ef51-11e8-8bb8-0a7084d0f53f.png)

All the images can be reproduced by running:
`python GenerateNeuroMLlite.py -interlaminar` for NeuroML/LEMS Analysis
`python Mejias-2016.py -analysis interlaminar_a` for the Python Analysis

JessyD commented:
@pgleeson I think this is now ready to be merged :)
